npm/rules.bzl

```diff
@@ -83,39 +83,37 @@ assemble_npm = rule(
 )
```

Comment:
Updating `deploy_npm` rule implementation to be backed by Kotlin script instead of Python